### PR TITLE
Gatsby starter theme added

### PR DESCRIPTION
**Pull request by rasif-sahl**

### CODE REVIEW

File: ported/gatsby-starter/assets/flashload.js [hunk 1]
```diff
@@ -0,0 +1,394 @@
+window.FlashLoad = (function() {
```
Comment (supunkavinda) [inline ported/gatsby-starter/assets/flashload.js:1]: You do not want to add `flashload.js`. I will remove it from Ghost-attila too. Flashload is a default asset.

`/assets/flashload.js` works for all blogs. :)

File: ported/gatsby-starter/config.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    primary: "#fff"
+    text: "#fff"
+    text_light: "#fff"
+    heading: "#fff"
+    heading_black: "#fff"
+    accent: "#fff"
+    background: "#282C35"
+
+font:
+  fontWeight_bold: "700"
```
Comment (supunkavinda) [inline ported/gatsby-starter/config.yaml:30]: Don't mix up cases. Use `font_weight_bold`

File: ported/gatsby-starter/templates/post.twig [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                    </section>
+                    <hr>
+                    <div class="bio">
+                        <p>Written by 
+                            <!-- -->
+                            <strong>{{ _blog.name }}</strong> 
+                            <!-- -->
+                            {{ _blog.description }}
+                            <!-- --> <!-- -->
+                            <a href="https://twitter.com/kylemathews">You should follow them on Twitter</a>
```
Comment (supunkavinda) [inline ported/gatsby-starter/templates/post.twig:72]: These links should be fixed

File: ported/gatsby-starter/templates/index.twig [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+    <main>
+        <div class="bio">
+            <p>Written by 
+                <!-- -->
+                <strong>{{ _blog.name }}</strong> 
+                <!-- -->
+                {{ _blog.description }}
+                <!-- --> <!-- -->
+                <a href="https://twitter.com/kylemathews">You should follow them on Twitter</a>
```
Comment (supunkavinda) [inline ported/gatsby-starter/templates/index.twig:45]: Link

Comment (rasif-sahl) [reply ported/gatsby-starter/templates/index.twig:45]: done

File: ported/gatsby-starter/templates/post.twig (hunk 3 above)
Comment (rasif-sahl) [reply ported/gatsby-starter/templates/post.twig:72]: done

File: ported/gatsby-starter/config.yaml (hunk 2 above)
Comment (rasif-sahl) [reply ported/gatsby-starter/config.yaml:30]: done

File: ported/gatsby-starter/assets/flashload.js (hunk 1 above)
Comment (rasif-sahl) [reply ported/gatsby-starter/assets/flashload.js:1]: done

**Pull request merged**